static/js/search.js

```diff
@@ -0,0 +1,18 @@
+var index = elasticlunr(function () {
+  this.addField('name');
```

Comment:
please make sure these fields use the names in https://github.com/sent-hil/sfsbook-1/blob/master/templates/resources.js.tmpl